src/components/Select.js

```diff
@@ -0,0 +1,183 @@
+import React from "react";
```

Comment:
The name of this file (`Select.js`) is a bit misleading, if it's notification specific, it should be clear from the name I think.

Reply:
Good point! I've renamed it to `NotificationSelect.js`. I don't have better naming on top of my head. The `Select` we define also enables select multiple options & select all, colored options, and creatable options. It should be easily extensible to other usages outside notifications. Probably we should name it to something that suggests its functionalities? I'm not sure. Let me know if you comes up with something!